File: Lógica/Lógic C#/Scripts/AttRevisão6/Models/Pessoa.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace AttRevisão6.Models {
    public class Pessoa {
        public string? Nome { get; private set; }
        public string? Sobrenome { get; private set; }
        public int Idade { get; private set; }


        public Pessoa SetNome(string nome){
            Nome = nome;
            return this;
        }
        public Pessoa SetSobrenome(string sobrenome){
            Sobrenome = sobrenome;
            return this;
        }
        public Pessoa SetIdade(int idade){
            Idade = idade;
            return this;
        }
        public Pessoa Apresentar(){
            Console.WriteLine($"Nome {Nome}, Sobrenome {Sobrenome}, Idade {Idade}");
            return this;
        }
        
        
    }
}

File: Lógica/Lógic Jav/src/POO/att9/Pessoa.java
package att9;

public class Pessoa {
    private String nome;
    private String sobrenome;
    private int idade;
    private String telefone;

    Pessoa(PessoaBuilder pessoaBuilder){
        this.nome = pessoaBuilder.nome;
        this.sobrenome = pessoaBuilder.sobrenome;
        this.idade = pessoaBuilder.idade;
        this.telefone = pessoaBuilder.telefone;
    }

    @Override
    public String toString(){
        return "Pessoa{" +
                "nome='" + nome + '\'' +
                ", sobrenome='" + sobrenome + '\'' +
                ", idade=" + idade +
                ", telefone='" + telefone + '\'' +
                '}';
    }

}
